src/Message/CancelSubscriptionRequest.php

```diff
@@ -13,6 +13,11 @@
  * or subscriptionReference is required.
  * - subscriptionReference: The gateway's identifier for the subscription to be canceled. Either
  * subscriptionId or subscriptionReference is required.
+ * - cancelReason: The reason that the subscription was canceled (optional). Possible values are
+ * documented by Vindicia here. Only the reason code needs to be specified:
+ * https://www.vindicia.com/documents/2500ProgGuideHTML5/Default.htm#ProgGuide/Canceling_AutoBills\
+ * _with.htm%3FTocPath%3DCashBox2500ProgGuide%7C5%2520Working%2520with%2520AutoBills%7C5.3%2520\
+ * Canceling%2520AutoBills%7C_____1
  *
```

Comment:
that url... 🤢

Reply:
The link you provide is for v25 of the Cashbox API but the version we actually use is v18